pitmutationmate/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,7 +3,7 @@
 
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-bin.zip
```

Comment:
why not up to 8.5 while were at it?

Reply:
The Problem is that when I used gradle 8.5 I got the the following error during the github jobs:
```
Exception is:
java.lang.NoClassDefFoundError: org/gradle/api/publish/ivy/internal/publication/DefaultIvyPublicationIdentity
```